### DISCUSSION TITLE
Time-Varying Depth Tutorial

### DISCUSSION THREAD

**Discussion by katherinecl** [Parcels-code/Parcels#1484] [Q&A]
Hello,

I am trying to implement time-varying depths as in 

https://docs.oceanparcels.org/en/latest/examples/tutorial_timevaryingdepthdimensions.html

And have a question about the data in the example, from SWASH. The SWASH data contains variables 'time varying depth' and 'time varying depth u', as well as 'z' and 'z_u'. Does anyone know what these variables actually relate to and what the difference is? 'Time varying depth u' is used as the depth for the example.

Thank you,
Katherine

**Comment by erikvansebille** [comment]
Dear @katherinecl, I _think_ this is because SWASH is on a C-grid, where the 'u' and 'v' points are not in the same location. But I haven't run SWASH myself so not entirely sure
